test_howdoi.py

```diff
@@ -43,37 +48,37 @@ def test_get_link_at_pos(self):
 
     def test_answers(self):
         for query in self.queries:
-            self.assertTrue(self.call_howdoi(query))
+            self.assertValidResponse(self.call_howdoi(query))
```

Comment:
Again, not sure if this is related

Reply:
Technically I think `assertValidResponse` is better than `assertTrue`, and this leaves the door open to improve `assertValidResponse`. The `assertTrue` was honestly me being lazy when I wrote the tests.